bitnami/common/templates/_capabilities.tpl

```diff
@@ -16,7 +16,7 @@ Return the target Kubernetes version
 {{- end -}}
 
 {{/*
-Return the appropriate apiVersion for policy.
+Return the appropriate apiVersion for podsecuritypolicy.
```

Comment:
This `policy/v1` applies to `PodDisruptionBudget` right? I think `PodSecurityPolicy` are not going out of beta

Reply:
We have a problem then... See:
- https://github.com/bitnami/charts/blob/master/bitnami/concourse/templates/web/podsecuritypolicy.yaml#L2
- https://github.com/bitnami/charts/blob/master/bitnami/concourse/templates/worker/podsecuritypolicy.yaml#L2
Bu we're also using it for PDB(s), see:
- https://github.com/bitnami/charts/blob/master/bitnami/haproxy/templates/pdb.yaml#L2
- https://github.com/bitnami/charts/blob/master/bitnami/minio/templates/gateway/pdb.yaml#L2
- https://github.com/bitnami/charts/blob/master/bitnami/redis/templates/pdb.yaml#L2

Reply:
I'll file an issue to address it!